CONTRIBUTING.rst

```diff
@@ -1,45 +1,75 @@
 Contributing
 ============
 
-For more information, please see the official docs at
-http://botocore.readthedocs.org/
+We work hard to provide a high-quality and useful SDK, and we greatly value
+feedback and contributions from our community. Whether it's a new feature,
+correction, or additional documentation, we welcome your pull requests. Please
+submit any `issues <https://github.com/boto/botocore/issues>`__
+or `pull requests <https://github.com/boto/botocore/pulls>`__ through GitHub.
+
+This document contains guidelines for contributing code and filing issues.
 
 Contributing Code
 -----------------
 
--  A good pull request:
+The list below are guidelines to use when submitting pull requests.
+These are the same set of guidelines that the core contributors use
+when submitting changes, and we ask the same of all community
+contributions as well:
+
+* The SDK is released under the
+  `Apache license <http://aws.amazon.com/apache2.0/>`__.
+  Any code you submit will be released under that license.
+* We maintain a high percentage of code coverage in our unit tests.  As
+  a general rule of thumb, code changes should not lower the overall
+  code coverage percentage for the project.  To help with this,
+  we use `coveralls <https://coveralls.io/r/boto/botocore>`__, which will
+  comment on every pull request with any changes in code coverage.
```

Comment:
"comment on every pull request with any changes in code coverage." -->  "comment on changes in code coverage for every pull request"

The code coverage sometimes does not change and it still comments.